ipam/manager.go

```diff
@@ -137,6 +138,18 @@ func (am *addressManager) restore() error {
 		}
 	}
 
+	if rebooted {
+		for _, as := range am.AddrSpaces {
```

Comment:
Add logs saying you are rehydrating the state, and what state is getting rehydrated.